readme.md

```diff
@@ -70,19 +70,25 @@ export class AppModule {}
 
 ### Sass
 
-Include the Kirby global styles in your app:
+Include the Kirby global styles in your app, e.g., in `src/styles.scss` :
```

Comment:
Excessive whitespace :) 
```suggestion
Include the Kirby global styles in your app, e.g., in `src/styles.scss`:
```

Reply:
I didn't add it. The hook(s) that run when pushing to remote does funky stuff to markdown files.

Reply:
I didn't add it. Perhaps added by `lint-staged`